language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/List.hs

```diff
@@ -124,7 +131,7 @@ getBuiltinFoldrList = do
           , Var () x
           ]
 
--- |  @foldl'ist@ as a PLC term.
+-- |  'foldl\'' as a PLC term.
```

Comment:
strangely different formatting

Reply:
`foldl'ist` was an autoreplace error. `'foldl\''` should be correct (not that I know haddock). @vmchale, is it fine?

Reply:
I guess I'm confused because the others all seem to use `@`s.

Reply:
`'x'` generates an clickable reference to the definition of `x`. If there is no definition, then @x@ renders just here on GitHub. I think.